benches/rt_multi_threaded.rs

```diff
@@ -57,19 +59,60 @@ fn spawn_many_remote_idle(b: &mut Bencher) {
     });
 }
 
-fn spawn_many_remote_busy(b: &mut Bencher) {
+fn spawn_many_remote_busy1(b: &mut Bencher) {
```

Comment:
maybe worth a comment here describing the different cases these benchmarks are exercising?